build/docker/client/Dockerfile

```diff
@@ -12,11 +12,13 @@ FROM redhat/ubi9-micro
 ARG VERSION
 ARG BUILD_DATE
 
-LABEL org.opencontainers.image.created ${BUILD_DATE}
-LABEL org.opencontainers.image.licenses Apache-2.0
-LABEL org.opencontainers.image.title Percona Monitoring and Management Client
-LABEL org.opencontainers.image.vendor Percona
-LABEL org.opencontainers.image.version ${VERSION}
+LABEL org.opencontainers.image.created=${BUILD_DATE} \
+      org.opencontainers.image.licenses=Apache-2.0 \
+      org.opencontainers.image.title="Percona Monitoring and Management Client" \
+      org.opencontainers.image.vendor="Percona, LLC" \
+      org.opencontainers.image.version=${VERSION} \
+      vendor="Percona, LLC" \
+      maintainer="Percona, LLC"
```

Comment:
IMO we need to update more labels. 

How about those:
- io.k8s.description
- io.k8s.display-name
- name
- summary
- url
- version

I believe we shouldn't leave them as they are now.

Reply:
How about skipping the `version` field since it will need to be updated with every release?

Reply:
If we consider the current value meaningful ("version": "9.5"), then we can leave it. 

BTW, what does it refer to? Release version of UBI?

Reply:
IMO the `version` tag should have the same value as `org.opencontainers.image.version`.